examples/shop-page/Components/ProductItem.lua

```diff
@@ -0,0 +1,69 @@
+local MarketplaceService = game:GetService("MarketplaceService")
+local Players = game:GetService("Players")
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+
+local Roact = require(ReplicatedStorage.Roact)
+
+local ProductItem = Roact.Component:extend("ProductItem")
+
+--[[
+	Props: {
+		image: the image displayed
+		price: how much robux the product cost
+		productId: the developer product id
+
+		-- Optional props
+		order: the index at which the item should be shown. If not
+		       provided, the price will be used as the index.
+	}
+]]
+
```

Comment:
We should use `validateProps` for this instead of writing a comment all on its lonesome!